WebUI/src/components/SettingsBasic.vue

```diff
@@ -61,14 +65,34 @@ const themeToDisplayName = (theme: Theme) => {
     }
 }
 
-
 const modelSettings = reactive<KVObject>(Object.assign({}, toRaw(globalSetup.modelSettings)));
+
+const modelSettingsWatcher = watchEffect(() => {
+  modelSettings.llm = globalSetup.modelSettings.llm;
+  modelSettings.sd_model = globalSetup.modelSettings.sd_model;
+  modelSettings.lora = globalSetup.modelSettings.lora;
+});
+
+onMounted(() => {
+  cancelModelSettingsChange();
+})
+
+onUnmounted(() => {
```

Comment:
in doubt, we should probably use onDecativated / onActivated, which are also called onMounted /onUnmounted, but works around some browser keep-alive cache, see https://vuejs.org/guide/built-ins/keep-alive#lifecycle-of-cached-instance